snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/config/SnomedCoreConfiguration.java

```diff
@@ -35,25 +31,24 @@
 public class SnomedCoreConfiguration {
 	
 	public static final String ELK_REASONER_ID = "org.semanticweb.elk.elk.reasoner.factory"; //$NON-NLS-1$
-	private static final String DEFAULT_REASONER = ELK_REASONER_ID;
+	public static final String DEFAULT_REASONER = ELK_REASONER_ID;
 	public static final int DEFAULT_MAXIMUM_REASONER_COUNT = 2;
 	public static final int DEFAULT_MAXIMUM_REASONER_RESULTS = 10;
 	public static final int DEFAULT_MAXIMUM_REASONER_RUNS = 1000;
 	
+	public static final String REASONER_EXCLUDE_MODULE_IDS = "reasonerExcludedModuleIds";
+	public static final String MAXIMUM_REASONER_RUNS = "maxReasonerRuns";
+	
 	@Min(1)
 	@Max(3)
 	private int maxReasonerCount = DEFAULT_MAXIMUM_REASONER_COUNT;
 	
-	@Min(1)
-	@Max(100)
-	private int maxReasonerResults = DEFAULT_MAXIMUM_REASONER_RESULTS;
-	
 	@Min(1)
 	@Max(1_000_000)
 	private int maxReasonerRuns = DEFAULT_MAXIMUM_REASONER_RUNS;
 	
-	@NotEmpty
-	private String defaultReasoner = DEFAULT_REASONER;
+	public static final String NAMESPACE_AND_MODULE_ASSIGNER = "namespaceModuleAssigner";
+	public static final String DEFAULT_NAMESPACE_AND_MODULE_ASSIGNER = "default";
```

Comment:
Move these to the class `com.b2international.snowowl.snomed.common.SnomedTerminologyComponentConstants` to be next to the other config keys and default values.